thorlcr/graph/thgraphmaster.cpp

```diff
@@ -649,6 +649,27 @@ void CMasterActivity::done()
 
 void CMasterActivity::updateFileReadCostStats()
 {
+     // Returns updates numDiskReads & readCost in the file attributes and returns the readCost
+    auto updateReadCosts = [](IDistributedFile *file, CThorStatsCollection &stats)
+    {
+        stat_type curDiskReads = stats.getStatisticSum(StNumDiskReads);
+        StringBuffer clusterName;
+        file->getClusterName(0, clusterName);
```

Comment:
This should be whichever cluster it was actually read from - it is likely to make a difference in the future.

Reply:
To revisit this: https://track.hpccsystems.com/browse/HPCC-30946